extras/help_generator/generate_help.py

```diff
@@ -131,5 +127,3 @@
 
             all_data = coll_data(keywords, documentation, num, helpdir, fname,
                                  sli_command_list)
-
-write_helpindex(helpdir)
```

Comment:
I have now implemented it as a seperate `COMMAND` in the `generate_help` target. I think that provides for a clearer separation of concerns.